package.json

```diff
@@ -26,6 +26,7 @@
     "format": "prettier --write \"**/*.{js,json,md}\"",
     "lint": "eslint .",
     "lint:fix": "npm run lint -- --fix",
+    "check:dependencies": "snyk test",
```

Comment:
`snyk test requires an authenticated account. Please run `snyk auth` and try again.`
='(